src/website/views/components/alert.pug

```diff
@@ -13,113 +13,219 @@ h4 Base
 .example.-mb--3
   .-p--1
     each type in ['success', 'danger', 'warning', 'info', 'muted']
```

Comment:
Examples in Base are going to be now the Alerts with icons, please change and remove the Icons section